code/data_structures/B_Tree/B_Tree.cpp

```diff
@@ -0,0 +1,101 @@
+// C++ implemntation of search() and traverse() methods
+#include<iostream>
+using namespace std;
+ 
```

Comment:
Could you implement the `insert` function using `splitChild()` and `delete` function? This will make a code complete 👍

This will make this good to be merged 👍

Reply:
ok sure 👍